01week/dataTypes.js

```diff
@@ -74,3 +74,28 @@ parseInt(string);
 // this does not work because no matter what you write in the prompt the computer reads it as a string
 var object = prompt("Write an examply of a datatype");
 document.write(typeof object);
+
+// homework
+
+// I did not even try prompt but I am assuming it wont work because it will return a string
+var firstNum = 25;
+var secondNum = 75;
+document.write(firstNum + secondNum);
+
+var thing1 = true;
+var thing2 = true;
+
+function willItRun {
+  if (thing1 = true && thing2 = true)
+  ; else
+};
+
+function howAboutNow {
+  if (thing1 = true || thing2 = true)
```

Comment:
This will cause a bug. You are re-assigning the value of thing1 and thing2 to the boolean true. You would need to use the === if you want to compare. However, I am not asking to evaluate if these two are the boolean true, I am asking if they evaluate to true. Also fix the function syntax and the if syntax.